src/app/[locale]/(default)/rules/layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { unstable_setRequestLocale } from 'next-intl/server';
+
+type RulesLayoutProps = {
+  children: React.ReactNode;
+  params: { locale: string };
+};
+
+export default function RulesLayout({
+  children,
+  params: { locale },
+}: RulesLayoutProps) {
+  unstable_setRequestLocale(locale);
+
+  return <>{children}</>;
```

Comment:
The layout is not needed if it is empty and not doing anything. But it should be doing something. I would insert the title here and move the page into a route group. Also you need a loading.tsx for the rulesPage, that can then be inside this route group